edx_rest_framework_extensions/tests/test_utils.py

```diff
@@ -38,10 +71,81 @@ def setUp(self):
         self.jwt = generate_jwt_token(self.payload)
 
     def test_decode_success(self):
-        self.assertDictEqual(jwt_decode_handler(self.jwt), self.payload)
+        """
+        Confirms that the format of the valid response from the token decoder matches the payload
+        """
+        self.assertDictEqual(utils.jwt_decode_handler(self.jwt), self.payload)
+
+    @ddt.data(*JWT_ISSUERS)
+    @override_settings(JWT_AUTH=JWT_AUTH_VALID_ISSUERS)
+    def test_decode_valid_token_multiple_valid_issuers(self, jwt_issuer):
+        """
+        Validates that a valid token is properly decoded given a list of multiple valid issuers
+        """
+
+        # Verify that each valid issuer is properly matched against the valid issuers list
+        # and used to decode the token that was generated using said valid issuer data
+        self.payload['iss'] = jwt_issuer['ISSUER']
+        token = generate_jwt_token(self.payload, jwt_issuer['SECRET_KEY'])
+        self.assertEqual(utils.jwt_decode_handler(token), self.payload)
 
-    def test_decode_error(self):
+    @override_settings(JWT_AUTH=JWT_AUTH_VALID_ISSUERS)
+    def test_decode_raises_invalid_token_error_invalid_token(self):
```

Comment:
This is pretty much the same as `test_decode_raises_invalid_token_error_no_issuers_matched`.